demo/index.html

```diff
@@ -0,0 +1,147 @@
+    <script src="https://unpkg.com/@playkit-js/playkit-js-dual-screen@canary/dist/playkit-dual-screen.js" type="text/javascript"></script>
+    <script src="https://unpkg.com/@playkit-js/playkit-js-youtube@canary/dist/playkit-youtube.js" type="text/javascript"></script>
+    <script src="https://unpkg.com/@playkit-js/call-to-action@canary/dist/playkit-call-to-action.js" type="text/javascript"></script>
+    <script src="https://unpkg.com/@playkit-js/playkit-js-timeline@canary/dist/playkit-timeline.js" type="text/javascript"></script>
+    <script src="https://unpkg.com/@playkit-js/playkit-js-kaltura-cuepoints@canary/dist/playkit-kaltura-cuepoints.js" type="text/javascript"></script>
+    <script src="https://unpkg.com/@playkit-js/playkit-js-ivq@canary/dist/playkit-ivq.js" type="text/javascript"></script>
+    <script src="https://unpkg.com/@playkit-js/playkit-js-kaltura-live@canary/dist/playkit-kaltura-live.js" type="text/javascript"></script>
+    <script src="https://unpkg.com/@playkit-js/hotspots@canary/dist/playkit-hotspots.js" type="text/javascript"></script>
+    <script src="https://unpkg.com/@playkit-js/slate@canary/dist/playkit-slate.js" type="text/javascript"></script>
+    <script src="./playkit-call-to-action.js" type="text/javascript"></script>
+  </head>
+  <body>
+    <div id="player-placeholder"></div>
+    <script type="text/javascript">
+      const config = {
+        log: {
+          level: 'DEBUG'
+        },
+        targetId: 'player-placeholder',
+        provider: {
+          env: {
+            serviceUrl: 'https://api.nvq2.ovp.kaltura.com/api_v3',
+            cdnUrl: 'https://api.nvq2.ovp.kaltura.com',
+            statsServiceUrl: 'https://analytics.rcqa3.kaltura.com',
+            liveStatsServiceUrl: 'https://livestats.nvq2.ovp.kaltura.com',
+            analyticsServiceUrl: 'https://analytics.nvq2.ovp.kaltura.com',
+            apiFeatures: {
+              entryRedirect: '1'
+            },
+            kmsHistoryProxyUrl: 'https://async-api.nvq2.ovp.kaltura.com/api_v3/'
+          },
+          partnerId: '242',
+          uiConfId: '23509762',
+          ks: 'djJ8MjQyfGDNO9E6UebFcKJbycYefhhWllSW1LgkHLrwMK6Uc0hIgJNEVPvWSwyl1gr0l1sMX5eqkMOirtANUMVFsLkQXB_wTnrIHD8nEJdWUZ06-vzYekJHaxfWSeyB4EVccaGBgQaq90lQMGXSYw6IiJcI_lZCGDP4YoVGX8SeO--Zfb8Sv8hXIMm5EGSMGqSVQRAm2G-zK4Q4_yVNGGHuxLiXgoQFoyZmKwl0T-uhVFfFXmyj_7VPOReJJTbfYFGNU4rx3ih_qax3jOUDo7hM3E02OHaJLkEA5xtGAittzFNGyqSuebPhzYSL01Rzn6nK5qbhQbeHs4cRflA4n4EloHGkF1i3l7QaQ9Rho6KMYCD1OitcaiFbZorYS5LRBQWp1HSSB0U5Rg0IR9moRzKwaVwS7qRJdoY0bWhv9KvvfTSofQLac7RAVwgPCkX3xo6LF9NVSWodfZ9l732LolRXrXNwBV8=',
+          log: {
+            level: 'ERROR'
+          }
+        },
+        ui: {
+          locale: 'en',
+          translations: {
+            en: {
+              slate: {
+                dismiss: 'Dismiss'
+              }
+            }
+          }
+        },
+        plugins: {
+          kava: {},
```

Comment:
why do you need all of the other plugins in a demo for cta ?

Reply:
to test it with plugins